tests/playground.ts

```diff
@@ -0,0 +1,150 @@
+import { LoginAsRootPage } from "../pom/login-as-root-page.js";
+import { ProductSelectionPage } from "../pom/product-selection-page.js";
+import fs from "fs";
+import path from "path";
+
+import puppeteer, { ElementHandle } from "puppeteer-core";
```

Comment:
this `{ ElementHandle }` is not needed, my IDE created automatically and I pushed accidentally.